File: src/Tokyo.Service/Models/Core/TimeNormTag.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace Tokyo.Service.Models.Core
{
    public class TimeNormTag
    {
        public int TimeNormTagId { get; set; }
        [Required, StringLength(255)]
        public string Name { get; set; } = "UNSET_TIMENORMTAG";
        public List<TimeNormTagMapping> TimeNormTagMappings { get; set; } = new List<TimeNormTagMapping>();
        [Required, StringLength(255)]
        public string Color { get; set; } = "red";
        [Required]
        public string TrackerStoreId { get; set; }
        public TrackerStore TrackerStore { get; set; }
    }
}

File: src/Tokyo.Client/Typewriter/TimeNormTag.ts


interface TrackerStore { TrackerStoreId: string; }
import {TimeNormTagMapping} from "./TimeNormTagMapping"; 
export class TimeNormTag { 
    TimeNormTagId: number;
    Name: string;
    TimeNormTagMappings: TimeNormTagMapping[] | undefined;
    Color: string;
    TrackerStoreId: string;
    TrackerStore: TrackerStore | undefined;
}